src/graphql.tsx

```diff
@@ -191,14 +191,18 @@ export default function graphql(
       return opts;
     }
 
+    function shouldSkip(props) {
+      return mapPropsToSkip(props) || (mapPropsToOptions(props) as QueryOptions).skip;
```

Comment:
this does end up calling mapPropsToOptions more often than before.  Dunno if that's a problem.